modules/oe_editorial_corporate_workflow/oe_editorial_corporate_workflow.module

```diff
@@ -28,3 +29,16 @@ function oe_editorial_corporate_workflow_menu_local_tasks_alter(&$data): void {
     }
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Force the Draft state on node edit forms.
+ */
+function oe_editorial_corporate_workflow_form_node_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if (isset($form['moderation_state']['widget'][0]['state']['#options']['draft'])) {
```

Comment:
With this code, and given the nature of the moderation form that disappears when the node is published, it becomes not possible anymore to put a node in the states "Archived" and "Expired".
This outlines the fact that we miss test coverage for the transitions in those states, we should introduce now these tests.

Reply:
We have not received clear requirements for those states. We don't know yet what those state will do and it's under design afaik.